draft-ietf-homenet-front-end-naming-delegation.txt

```diff
@@ -1168,21 +1188,11 @@ Internet-Draft                public-names                     July 2021
    occur, for example, when the ISP provides the Homenet Domain as well
    as the DOI.
 
-
-
-
-
-
-Migault, et al.          Expires 14 January 2022               [Page 21]
-
-Internet-Draft                public-names                     July 2021
-
-
-   In this case, the HNA may be authenticated by the physical link
-   layer, in which case the authentication of the HNA may be performed
-   without additional provisioning of the HNA.  While this may not be so
-   common for the Public Homenet Zone, this situation is expected to be
-   quite common for the Reverse Homenet Zone.
+   In this case, the HNA may be authenticated by the physical link
+   layer, in which case the authentication of the HNA may be performed
+   without additional provisioning of the HNA.  While this may not be so
+   common for the Public Homenet Zone, this situation is expected to be
+   quite common for the Reverse Homenet Zone.
```

Comment:
why is it more common for reverse homenet? 
Actually it will be nice if you can start the paragraph with how reverse homenet zone will be used and built (its not covered in channels).
also the terminology uses "Homenet Reverse Zone"